suite-native/app/src/App.tsx

```diff
@@ -58,12 +62,25 @@ const AppComponent = () => {
             } catch (error) {
                 Alert.alert('Error', error?.message ?? 'Unknown error');
                 console.error(error.message);
+            } finally {
+                // Tell the application to render
+                setIsAppReady(true);
             }
         };
         initActions();
     }, [dispatch, isConnectInitialized]);
 
-    if (!isConnectInitialized) {
+    useEffect(() => {
+        const checkSplashScreenHide = async () => {
+            if (isAppReady) {
+                await SplashScreen.hideAsync();
+            }
+        };
+
+        checkSplashScreenHide();
+    }, [isAppReady]);
+
+    if (!isConnectInitialized && !isAppReady) {
```

Comment:
Please remove `isConnectInitialized ` from this check, we want to show app even if connect fail to init. Better than blank white screen.

Reply:
6eb52e06277547303899f873a87bcd4f9fcddf50